src/metrics/scenario.jl

```diff
@@ -203,12 +191,12 @@ metrics::Vector{ADRIA.metrics.Metric} = [
 outcomes = ADRIA.metrics.scenario_outcomes(rs, metrics)
 ```
 """
-function scenario_outcomes(rs::ResultSet, metrics::Vector{<:Metric})::NamedDimsArray
+function scenario_outcomes(rs::ResultSet, metrics::Vector{<:Metric})::YAXArray
     n_timesteps = length(timesteps(rs))
     n_scenarios = size(rs.inputs, 1)
     n_metrics = length(metrics)
 
-    scen_outcomes = NamedDimsArray(
+    scen_outcomes = DataCube(
         zeros(n_timesteps, n_scenarios, n_metrics);
```

Comment:
```suggestion
    scen_outcomes = ZeroDataCube(;
        T=Float,
```
Is this a place to use the ZeroDataCube constructor?